redish/libs/redish-frontend/authentication/feature/src/lib/authentication-provider/auth-provider.tsx

```diff
@@ -0,0 +1,40 @@
+import { useAuthenticationCore } from '@redish-frontend/authentication-data-access';
+import { ReactNode, createContext, useContext, useMemo } from 'react';
+
+/**
+ * Auth provider as in https://dev.to/sanjayttg/jwt-authentication-in-react-with-react-router-1d03
+ */
+
+export const AuthContext = createContext<{
+  token: string | null;
+  setToken: (newToken: string | null) => void;
+} | null>(null);
+
+type AuthProviderProps = { children: ReactNode };
+
+export const AuthProvider = ({ children }: AuthProviderProps) => {
+  const { token, setToken } = useAuthenticationCore();
+
+  // Memoized value of the authentication context
+  const contextValue = useMemo(
+    () => ({
+      token,
+      setToken,
+    }),
+    [token]
```

Comment:
For me there is a note here: "React Hook useMemo has a missing dependency: 'setToken'"

Reply:
applied